web/elm/src/Resource/Resource.elm

```diff
@@ -789,12 +805,12 @@ permalink : List Concourse.VersionedResource -> Page
 permalink versionedResources =
```

Comment:
This is unused

```
Messages:

- src/Resource/Resource.elm

  > Unused top level definition `permalink` at ((805,1),(805,10))

error Command failed with exit code 1.                                         

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Did a quick check:
```
┌─[issue/5977*][~/workspace/concourse/concourse/web/elm]
└─▪ ag 'permalink'
src/Job/Job.elm
302:permalink : List Concourse.Build -> Page
303:permalink builds =
379:            permalink paginatedBuilds.content

src/Resource/Resource.elm
804:permalink : List Concourse.VersionedResource -> Page
805:permalink versionedResources =
```

Reply:
Fixed!